packages/modal/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@telegraph/modal",
+  "version": "0.0.10",
```

Comment:
Should this be `0.0.1` or are new components just matching the rest of telegraph?